### PR TITLE
Add telemetry

### PR DESCRIPTION
**Pull request by zmssp**
* Add telemetry for docker, subscription
* Modify the event name to AzurePlugin.eclipse/opstart,   AzurePlugin.Intellij/opstart

### CODE REVIEW

File: Utils/azuretools-core/src/com/microsoft/azuretools/telemetry/AppInsightsClient.java [hunk 1]
```diff
@@ -186,7 +186,12 @@ private static void initTelemetryManager() {
         try {
             TelemetryManager.getInstance().setCommonProperties(buildProperties("", new HashMap<>()));
             TelemetryManager.getInstance().setTelemetryClient(TelemetryClientSingleton.getTelemetry());
-            TelemetryManager.getInstance().setEventNamePrefix(configuration.eventNamePrefix());
+            // The eclipse eventName prefix is AzurePlugin.Eclipse. We should remove the last "."
```
Comment (Flanker32) [inline Utils/azuretools-core/src/com/microsoft/azuretools/telemetry/AppInsightsClient.java]: What don't we change the event name to `AzurePlugin.Eclipse`?

**Comment by Flanker32** [comment]
Should we also change telemetries for webapp?

File: Utils/azuretools-core/src/com/microsoft/azuretools/telemetry/AppInsightsClient.java (hunk 1 above)
Comment (zmssp) [reply Utils/azuretools-core/src/com/microsoft/azuretools/telemetry/AppInsightsClient.java]: Add a new method eventName()

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/actions/GithubSurveyAction.java [hunk 2]
```diff
@@ -1,34 +1,38 @@
...
+import com.microsoft.azuretools.telemetrywrapper.EventUtil;
+import com.microsoft.intellij.feedback.GithubIssue;
+import com.microsoft.intellij.feedback.NewGithubIssueAction;
+import com.microsoft.intellij.feedback.ReportableSurvey;
+
+
+public class GithubSurveyAction extends NewGithubIssueAction {
+    public GithubSurveyAction() {
+        super(new GithubIssue<>(new ReportableSurvey("User feedback")), "Provide Feedback...");
+        EventUtil.logEvent(EventType.info, TelemetryConstants.FEEDBACK, TelemetryConstants.REPORT_ISSUES, null);
```
Comment (Flanker32) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/actions/GithubSurveyAction.java]: I think we may create a new service called SYSTEM which contains Toolkits operation, but FEEDBACK is also fine.

**Pull request merged**